Resources/Private/Form/StaticText.html

```diff
@@ -1,6 +1,7 @@
+{namespace form=TYPO3\Form\ViewHelpers}
```

Comment:
@gerhard-boden don't worry, it was not easy to spot - I only discovered it by chance ;)